README.md

```diff
@@ -347,16 +359,55 @@ translates to:
 * TooManyStatements#max_statements is 10 in "app/controllers"
 * TooManyStatements#max_statements is 5 everywhere else
 
-For more details please check out the [Basic Smell Options](docs/Basic-Smell-Options.md)
-which are supported by every smell type. As you can see above, certain smell
-types offer a configuration that goes beyond that of the basic smell options, for instance
+Every smell detector supports our [Basic Smell Options](docs/Basic-Smell-Options.md). As you can see above,
+certain smell types offer a configuration that goes beyond that of the basic smell options, for instance
 [Data Clump](docs/Data-Clump.md).
 All options that go beyond the [Basic Smell Options](docs/Basic-Smell-Options.md)
 are documented in the corresponding smell type /docs page (if you want to get a quick overview over all possible
-configurations you can also check out [the `docs/default.reek` file in this repository](docs/defaults.reek).
+configurations you can also check out [the `defaults.reek.yml` file in this repository](defaults.reek.yml).
```

Comment:
Is it your intention to move `defaults.reek.yml` out of `docs/` again?

Reply:
Not at all, can't recall how this happened, reverted ;)